styleguide/components/41-organisms/feedback-form/feedback-form.twig

```diff
@@ -0,0 +1,39 @@
+<div class="feedback-form {{ modifier }}">
+  <h3><button>{{ 'Can we improve something about this page?' }}<i class="icon-chevron-down" aria-hidden="true"></i></button></h3>
```

Comment:
Suggest already adding aria-exanded="true" (since that is it's state for now)
and aria-controls=[id of the expanded element]

Reply:
I'll add it for now... but once we add the JS functionality for these types of component the default will probably be aria-expanded="false" instead of true.